### ISSUE TITLE
Black Market layout not working as intended (possibly?)

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#956]
https://clips.twitch.tv/CloudyConcernedCodSoBayed-IEetVQyHsATTpMhB

just doesn't ever reroll no matter what lol, barely any point in a restock box machine here then

**Comment by Rchardon** [comment]
In shops reroll machines are meant to reroll buyable things, not pedestals.

**Comment by Lonrite** [comment]
> In shops reroll machines are meant to reroll buyable things, not pedestals.

yeah I know, just seems odd that there's a restock machine still in this layout - feels like it was either wasn't tested or put much thought into it or was meant to be a troll

**Comment by ZMBanshee** [comment]
It's a rare layout from Afterbirth+. I've known about this for a long time and yeah, hard to say what the purpose of it was. 

But upon looking through the layouts, I see that there's also a rare "bad" layout right next to it with only 1 item on sale and some flies. My hunch is that Edmund wanted the layout in your post to be a "good" layout where you can get stuff for free and reroll them, as Afterbirth+ introduced a lot of these "rare good" and "rare bad" type layouts.

**Comment by Lonrite** [comment]
makes sense
